js/app.js

```diff
@@ -0,0 +1,36 @@
+request = window.superagent;
+
+var react = angular.module('react', []);
+
+react.controller('MainController', function($scope) {
+
+  $scope.getWord = function() {
+
+    var url = 'http://api.wordnik.com:80/v4/words.json/randomWords?hasDictionaryDef=true&minCorpusCount=0&maxCorpusCount=-1&minDictionaryCount=1&maxDictionaryCount=-1&minLength=10&maxLength=-1&limit=1&api_key=a2a73e7b926c924fad7001ca3111acd55af2ffabf50eb4ae5';
+
+    request.get(url, function(res){
+
+      $scope.$apply(function() {
+
+        $scope.word = res.body[0].word;
+
+        var definitionURL = 'http://api.wordnik.com:80/v4/word.json/'+ $scope.word +'/definitions?limit=200&includeRelated=true&useCanonical=false&includeTags=false&api_key=a2a73e7b926c924fad7001ca3111acd55af2ffabf50eb4ae5';
+
+        request.get(definitionURL, function(res){
+
+          $scope.definition = res.body[0].text;
+
+        });
+
+      });
+
+    });
+
+  };
```

Comment:
This seems like a perfect use-case for a [Service](https://docs.angularjs.org/guide/services), keeping the data-fetching logic separated from the data passing part.